src/hooks/useWalletPDNTs/useWalletPDNTs.tsx

```diff
@@ -149,9 +151,26 @@ export function useWalletPDNTs(ids: ArweaveTransactionID[]) {
         className: 'white',
         ellipsis: true,
         render: (val: string) =>
-          `${val.slice(0, isMobile ? 2 : 6)}...${val.slice(
-            isMobile ? -2 : -6,
-          )}`,
+          val.length == 43 ? (
```

Comment:
we should be setting `targetID` to `None` or `N/A` in the table as opposed to `...` - so let's change that based on figma and do the check here when it's not equal to that value instead of just checking the length